lms/djangoapps/organizations/views.py

```diff
@@ -23,6 +23,24 @@
 from .serializers import OrganizationSerializer, BasicOrganizationSerializer
 
 
+def _get_course_response_data(course_descriptor, course_enrollments):
```

Comment:
I think instead of writing a private method to serialize data we should write a serializer class which should be a sub-class of `api_manager.courses.CourseSerializer`.